src/app/dts/template-editor/template-editor.component.html

```diff
@@ -0,0 +1,26 @@
+<form [formGroup]="templateEditor">
+  <mat-form-field>
+    <mat-label>Template Key</mat-label>
+    <input matInput type="text" formControlName="templateKey" [readonly]="true">
+  </mat-form-field>
+  <mat-form-field class="full-width">
+    <mat-label>Template Id</mat-label>
+    <input matInput type="text" formControlName="templateId" [readonly]="true">
+  </mat-form-field>
+  <mat-form-field>
+    <mat-label>Author</mat-label>
+    <input matInput type="text" formControlName="author" [readonly]="true">
+  </mat-form-field>
+  <ckeditor formControlName="templateData"
+    [config]="{
+      toolbar: [
```

Comment:
I'd move all that config object into your Typescript code instead of defining it in the HTML template.

Reply:
Fixed.  Implemented recommended change.